docs/intro/tutorial.rst

```diff
@@ -13,8 +13,8 @@ a website that lists books from famous authors.
 This tutorial will walk you through these tasks:
 
 1. Writing a :ref:`spider <scrapy:topics-spiders>` to crawl a site and extract data
-2. Configuring Scrapy project to include Injection Middleware
-3. Separating extraction logic from spider
+3. Separating extraction logic from the spider
```

Comment:
3 -> 2

Reply:
It is better to use `1.` for all items in an enumerated list. It will be auto-generated with the right numbers and avoids problems like this when you include new items to the list.

Reply:
Thanks, fixed!